server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -251,7 +262,9 @@ private FormTag renderSearchForm(
                     downloadButton,
                     makeSvgTextButton("Filter", Icons.FILTER_ALT)
                         .withClass(ButtonStyles.SOLID_BLUE_WITH_ICON)
-                        .withType("submit")));
+                        .withType("submit"),
+                    a().withHref(redirectUrl)
```

Comment:
@zarifzm - i recently learnt adding a button tag within an a tag can cause weird behaviour in html5. My suggestion is remove the button tag and move its style, text and href to a tag. So it will be a("Clear").withHref().withClass()